projects/plugins/jetpack/modules/masterbar/admin-menu/class-admin-menu.php

```diff
@@ -126,6 +132,35 @@ public function add_stats_menu() {
 		add_menu_page( __( 'Stats', 'jetpack' ), __( 'Stats', 'jetpack' ), 'view_stats', 'https://wordpress.com/stats/day/' . $this->domain, null, 'dashicons-chart-bar', 3 );
 	}
 
+	/**
+	 * Adds WordPress.com menu.
+	 */
+	public function add_wpcom_menu() {
+		if ( get_option( 'wpcom_is_staging_site' ) ) {
```

Comment:
Why is this needed?

Reply:
I think to maintain the current behavior for the staging site since there isn't an Upgrade menu
![staging site](https://github.com/Automattic/jetpack/assets/6586048/00a5cd13-fcc0-4330-b4da-a92813d76caf)

I think bringing back the WordPress.com menu can be addressed in a separate PR.

Reply:
Yep. I didn't show the Upgrades menu for the staging site, so I'd prefer to keep it hidden for now. When we have more and more items in the WordPress.com menu, we can remove this check.

Reply:
TIL about this, thanks for explaining!